pkg/controller/master-controller-manager/rbac/rbac_controller_aggregator.go

```diff
@@ -161,7 +161,7 @@ func New(metrics *Metrics, mgr manager.Manager, seedManagerMap map[string]manage
 			kind:      "Secret",
 			namespace: "kubermatic",
 			shouldEnqueue: func(obj metav1.Object) bool {
-				// do not reconcile secrets without "sa-token" and "credential" prefix
+				// do not reconcile secrets without "sa-token", "credential" and "kubeconfig-external-cluster" prefix
```

Comment:
In the future I would tend to avoid repeating the list of prefixes here, as it can get quickly out of sync and convey false impressions about what we do here. Likewise, in the future, someone might want to refactor `shouldEnqueueSecret(string)` into `shouldEnqueueSecret(metav1.Object)`, so we can avoid this tiny wrapper closure alltogether.